index.html

```diff
@@ -215,23 +135,24 @@ <h4 class="skills">Skills</h4>
         like to chat please get in touch.
       </p>
     </div>
-    <form id="contactForm" action="https://formspree.io/f/xzbqnzke" method="POST">
+    <form  action="https://formspree.io/f/xzbqnzke" method="POST">
       <ul class="contact-fields">
         <li>
           <label for="fullName" hidden>Full Name</label>
-          <input type="text" id="full-name" name="fullName" placeholder="Full name" maxlength="30" required />
+          <input type="text" id="full-name" class="name-input" name="fullName" placeholder="Full name" maxlength="30" required />
```

Comment:
- [ ] [OPTIONAL] It will be better 😃 if you let the user be able to write the full name in the field of full name as I see when I try to click on the full name field and write something in the full name field it does not allow me to write anything, so please try to fix that to let the user write his full name and communicate with you 💯😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
| --- |
|   ![Recording 2023-06-30 at 11 33 27](https://github.com/Zuwairanajma/Portfolio-Projects/assets/83360636/48db4ae8-661a-44c4-bd92-e788871fa49c)  |